apps/directory/src/components/configuration/JsonEditor.vue

```diff
@@ -25,7 +25,7 @@
           Upload config
         </button>
       </div>
-      <small class="ml-auto">To format your file press ctrl + f</small>
+      <small class="ml-auto">To format your file press ctrl + alt + f</small>
```

Comment:
For good semantic practices, the keyboard element (`<kbd>`) should be used.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd